client/samples/bin/launcher.rb

```diff
@@ -5,6 +5,7 @@
 @samples = {
     'connection' => 'lib/sample/workflow/connection_workflow.rb',
     'tagging' => 'lib/sample/workflow/tagging_workflow.rb',
+	'list_vms' => 'lib/sample/vcenter/vm/list_vms.rb',
```

Comment:
Can you fix the indentation on this? It's not the correct indentation anyway since Ruby should be indented two spaces (not four), but I'd like this to at least be consistent with what's existing.